rmgpy/qm/mopac.py

```diff
@@ -45,100 +45,72 @@
 class Mopac(object):
     """
     A base class for all QM calculations that use MOPAC.
-    
+
     Classes such as :class:`MopacMol` will inherit from this class.
     """
 
-    inputFileExtension = '.mop'
-    outputFileExtension = '.out'
-
-    executablesToTry = ('MOPAC2016.exe', 'MOPAC2012.exe', 'MOPAC2009.exe', 'mopac')
+    inputFileExtension = ".mop"
+    outputFileExtension = ".out"
 
-    for exe in executablesToTry:
-        try:
-            executablePath = distutils.spawn.find_executable(exe)
-        except:
-            executablePath = None
-        if executablePath is not None:
-            break
-    else:  # didn't break
-        logging.debug("Did not find MOPAC on path, checking if it exists in a declared MOPAC_DIR...")
-        mopacEnv = os.getenv('MOPAC_DIR', default="/opt/mopac")
-        for exe in executablesToTry:
-            executablePath = os.path.join(mopacEnv, exe)
-            if os.path.exists(executablePath):
-                break
-        else:  # didn't break
-            executablePath = os.path.join(mopacEnv, '(MOPAC 2009 or 2012 or 2016)')
+    try:
+        executablePath = distutils.spawn.find_executable("mopac")
+    except:
+        logging.debug(
+            "Did not find MOPAC on path, checking if it exists in a declared MOPAC_DIR..."
+        )
+        mopacEnv = os.getenv("MOPAC_DIR", default="/opt/mopac")
+        executablePath = os.path.join(mopacEnv, "mopac")
+        if not os.path.exists(executablePath):
+            executablePath = os.path.join(mopacEnv, "(MOPAC 2009 or 2012 or 2016)")
```

Comment:
This entire block does not make sense to me, I left it in to spark some discussion.

I think in the previous versions we may have allowed users to specify a different MOPAC executable and then retrieved it from there, but this isn't something we are interested in now since we ship with the latest release.

I will just remove this. It's dead and untested anyway.